CONTRIBUTING.md

```diff
@@ -277,17 +274,17 @@ Github [labels](https://github.com/eclipse-iceoryx/iceoryx/labels) are used to g
 
 If one of the rules is not followed, a rationale is added in the following manner:
 
-With a comment in the same line:
+Either with a comment in the same line:
 
 ```cpp
-    *mynullptr = foo; // PRQA S 4242 # Short description why
+*mynullptr = foo; // PRQA S 4242 # Short description why
```

Comment:
This is for Helix QAC right? Do we actually do/use this right now?

But in general it is always similar, a deviation rationale comment with special syntax is added.

Reply:
I think it's for Helix QAC and yes, in general the deviation comment should always look quite similar. @mossmaurice do we use it and shall state it here?

Reply:
@FerdinandSpitzschnueffler A grep revealed that it is used currently at 33 places in the code. For now, I would leave it as it is. In the long-run I would prefer to use vendor-neutral syntax based on C++ attributes. After switching to Axivion with #409, chances are high we might need to switch to the Axivion syntax for practical reasons. Axivion understands both syntaxes, but it could be cumbersome to use.